lib/language_server/protocol/interface/text_document_content_change_event.rb

```diff
@@ -6,10 +6,10 @@ module Interface
       # it is considered to be the full content of the document.
       #
       class TextDocumentContentChangeEvent
-        def initialize(range:, range_length: nil, text:)
+        def initialize(range: nil, range_length: nil, text:)
```

Comment:
https://github.com/mtsmfm/language_server-protocol-ruby/issues/48 can be fixed by this change

Reply:
Yes, this is the type we want. 👍